### PR TITLE
Support footer info

### PR DESCRIPTION
**Pull request by haoxusci**

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -149,6 +149,25 @@
           >
         </div>
       </div>
+      <div
+        class="columns is-moible is-centered"
+        v-if="siteConfig.footer_covid19ai"
```
Comment (oeway) [inline src/views/Home.vue]:
```suggestion
        v-if="siteConfig.footer"
```

File: src/views/Home.vue [hunk 2]
```diff
@@ -149,6 +149,25 @@
...
+          class="column is-one-quarter"
+        >
+          <b-tooltip :label="item.tooltip" position="is-top"
+            ><a :href="item.url" target="_blank">
+              <img :src="item.logo" style="height: 55px;" />
+            </a>
+          </b-tooltip>
+        </div>
+      </div>
+      <div style="text-align: center" v-if="siteConfig.footer_covid19ai_note">
```
Comment (oeway) [inline src/views/Home.vue]:
```suggestion
      <div style="text-align: center" v-if="siteConfig.footnote">
```

File: src/views/Home.vue [hunk 3]
```diff
@@ -149,6 +149,25 @@
...
+        <p>{{ siteConfig.footer_covid19ai_note }}</p>
```
Comment (oeway) [inline src/views/Home.vue]:
```suggestion
        <p>{{ siteConfig.footnote }}</p>
```

File: src/views/Home.vue [hunk 4]
```diff
@@ -149,6 +149,25 @@
           >
         </div>
       </div>
+      <div
+        class="columns is-moible is-centered"
+        v-if="siteConfig.footer_covid19ai"
+      >
+        <div
+          v-for="item in siteConfig.footer_covid19ai"
```
Comment (oeway) [inline src/views/Home.vue]:
```suggestion
          v-for="item in siteConfig.footer"
```

**Comment by oeway** [comment]
I made some suggestions to make remove it more general, since I think we don't need to make it specific to one website.

File: src/views/Home.vue (hunk 1 above)
Comment (haoxusci) [reply src/views/Home.vue]: done

File: src/views/Home.vue (hunk 2 above)
Comment (haoxusci) [reply src/views/Home.vue]: done

File: src/views/About.vue [hunk 5]
```diff
@@ -9,6 +9,25 @@
       </div>
     </section>
     <img class="background-img" :src="siteConfig.background_image" />
+    <br />
+
+    <div class="columns is-moible is-centered" v-if="siteConfig.footer">
```
Comment (MartinHjelmare) [inline src/views/About.vue]: Is `moible` a typo?

Comment (haoxusci) [reply src/views/About.vue]: right, it is a typo. resolved

**Review by oeway — APPROVED** [review]
Thanks @haoxusci Looks good to me.

File: src/views/Home.vue [hunk 6]
```diff
@@ -134,21 +134,23 @@
     <br />
 
     <footer class="footer">
-      <div class="columns is-multiline">
+      <div class="columns is-moible is-centered" v-if="siteConfig.footer">
```
Comment (MartinHjelmare) [inline src/views/Home.vue:137]: There's another typo here.

**Pull request merged**